src/coprocessor/codec/error.rs

```diff
@@ -66,6 +67,17 @@ quick_error! {
 }
 
 impl Error {
+    pub fn handle_invalid_time_error(ctx: &mut EvalContext, err: Error) -> Option<Error> {
+        if err.is_invalid_time_format() {
+            return Some(err);
+        }
+        if ctx.cfg.strict_sql_mode && (ctx.cfg.in_insert_stmt || ctx.cfg.in_update_or_delete_stmt) {
```

Comment:
Good job! Seems that other date time expressions need to use this function as well inside. Would you also like to propose other PRs to fix them?

Reply:
Yes, I will fix them after merge this pr and i will add other handle_invalid_xxxx_error like Tidb.